### ISSUE TITLE
Using dns-prefetch is broken

### ISSUE THREAD

**Issue by JBerendes** [stevenvachon/broken-link-checker#130]
`<link rel="dns-prefetch" href="//fonts.googleapis.com">`

Takes the current url and tacks the route on to the dns-prefetch url

example:
`http://randomexamplething.com/my/route`

blc will attempt to check the following and get a 404:
`├─BROKEN─ https://fonts.googleapis.com/my/route/ (HTTP_404)`

**Comment by XhmikosR** [comment]
Yup, I hit this too. Not sure if it happens with the other hints' tags too, like preconnect.

**Comment by zwadl** [comment]
Stumbled upon this too! 
Seems to happen with alle urls that have no protocoll in front. 

//www.whateverdomain.com then will be checked as http://www.whateverdomain.com/my/route/
